app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -73,7 +73,7 @@ class UseOurAppCta(
     @StringRes val okButton: Int = R.string.useOurAppDialogButtonText,
     @StringRes val cancelButton: Int = R.string.useOurAppDialogCancelButtonText,
     override val ctaId: CtaId = CtaId.USE_OUR_APP,
-    override val shownPixel: Pixel.PixelName? = AppPixelName.USE_OUR_APP_DIALOG_SHOWN,
```

Comment:
This pixel was causing way too many issues so we decided with product to remove it and rely on the notification pixel instead.